packages/js-sync-with-portabl/src/lib/fetchRetries.ts

```diff
@@ -0,0 +1,27 @@
+export const MAX_RETRIES = 3;
+
+export async function fetchRetries<T>(
```

Comment:
Nit: We could name this something more ambiguous than "fetch" -- i feel since we are executing async functions with retries maybe something like `withRetries()` is a more suitable name.